OracleGoldenGate/Oracle/12.2.0.1.1/runSQL.sql

```diff
@@ -0,0 +1,32 @@
+set echo on
+set serveroutput on
+SPOOL /tmp/runSQL.log
+alter system set enable_goldengate_replication=true;
+--alter system set log_archive_dest_1='location=/opt/datafile/ora12102i/arch';
+alter system set processes=300 scope=spfile;
+alter system set sessions=500 scope=spfile;
+shutdown immediate;
+startup mount;
+alter database archivelog;
+alter database open;
+alter system switch logfile;
+archive log list;
+--select * from v$encryption_wallet;
+alter system set job_queue_processes=1000;
+alter system set streams_pool_size=200m;
+alter system set shared_pool_size=500m;
+alter database add supplemental log data;
+--drop tablespace GGATE including contents and datafiles;
+create tablespace GGATE datafile '/u01/app/oracle/oradata/ggate01.dbf' size 1G;
+drop user ggate cascade;
+create user ggate identified by ggate default tablespace GGATE temporary tablespace TEMP quota unlimited on GGATE account unlock;

```

Comment:
It is changed and I am testing it.  It was my QA image and was directed to use oracle/database.  Thanks for your help, I incorporated the changes above, specially README.,md.  I wasn't the writer.